### ISSUE TITLE
test stk_param_init FAIL on octave 8.2.1 (and 9.0.0)

### ISSUE THREAD

**Issue by dasergatskov** [stk-kriging/stk#38]
```
***** test
 model = stk_model (@stk_materncov_iso);
 [param0, model.lognoisevariance] = stk_param_init (model, xi, zi, BOX);
 model.param = stk_param_estim (model, xi, zi, param0);
 zp = stk_predict (model, xi, zi, xt);
 assert (max ((zp.mean - zt) .^ 2) < 1e-3)
warning: sqp: QP subproblem is non-convex and unbounded
warning: called from
    sqp at line 436 column 9
    stk_minimize_boxconstrained at line 49 column 14
    stk_param_estim_optim at line 70 column 23
    stk_param_estim at line 139 column 15
    __test__ at line 5 column 14
    test at line 682 column 11

!!!!! test failed
operator *: nonconformant arguments (op1 is 3x6, op2 is 0x0)
shared variables     xi =

      <class stk_factorialdesign>

    zi =

      <class stk_dataframe>

    BOX =

      -1
       1

    xt =

      <class stk_factorialdesign>

    zt =

      <class stk_dataframe>
```

**Comment by dasergatskov** [comment]
It looks like there is a regression in sqp function in octave 8.2+:
https://savannah.gnu.org/bugs/?64302

**Comment by jbect** [comment]
Thank you @dasergatskov :+1:  I will look into it.

**Comment by jbect** [comment]
Closed by commit ecf6be8d4a94a6ddd28cebcff44308287ce419c8
